edgar/filing.py

```diff
@@ -18,34 +18,49 @@ class Statements:
     # To resolve "could not find anything for ShortName..." error, likely need
     # to add the appropriate ShortName from the FilingSummary.xml here.
     # TODO: perhaps add guessing/best match functionality to limit this list
-    income_statements = ['consolidated statements of income',
-                    'consolidated statements of operations',
-                    'consolidated statement of earnings',
-                    'condensed consolidated statements of income (unaudited)',
-                    'condensed consolidated statements of income',
-                    'condensed consolidated statements of operations (unaudited)',
-                    'condensed consolidated statements of operations',
-                    'condensed consolidated statement of earnings (unaudited)',
-                    'condensed consolidated statement of earnings',
-                    'condensed statements of income',
-                    'condensed statements of operations',
-                    'condensed statements of operations and comprehensive loss'
-                    ]
-    balance_sheets = ['consolidated balance sheets',
-                    'consolidated statement of financial position',
-                    'condensed consolidated statement of financial position (current period unaudited)',
-                    'condensed consolidated statement of financial position (unaudited)',
-                    'condensed consolidated statement of financial position',
-                    'condensed consolidated balance sheets (current period unaudited)',
-                    'condensed consolidated balance sheets (unaudited)',
-                    'condensed consolidated balance sheets',
-                    'condensed balance sheets'
-                    ]
-    cash_flows = ['consolidated statements of cash flows',
-                    'condensed consolidated statements of cash flows (unaudited)',
-                    'condensed consolidated statements of cash flows',
-                    'condensed statements of cash flows'
-                    ]
+    income_statements = [
+        'consolidated statements of income',
+        'consolidated statements of operations',
+        'consolidated statement of earnings',
+        'condensed consolidated statements of income (unaudited)',
+        'condensed consolidated statements of income',
+        'condensed consolidated statements of operations (unaudited)',
+        'condensed consolidated statements of operations',
+
+        'interim condensed consolidated statements of operations',
```

Comment:
The main functional change in this PR appears to be updates to the constants in the Statements class for “interim” statements. Could you please add tests for these so that i have clear examples of what prompted this need and so we can be confident that the changes have any negative consequences? In initially developing those constants i recall that certain names didn’t actually correspond to the 10-q and 10-k reports that we’re concerned with, so i prefer we be safe.